lib/vimrunner/server.rb

```diff
@@ -138,7 +141,7 @@ def new_client
     #
     # Returns an Array of String server names currently running.
     def serverlist
-      execute([executable, "--serverlist"]).split("\n")
+      execute([executable, "--serverlist"]).split(/\r?\n/)
```

Comment:
We might be able to use `$/` here which should return the platform-dependent end-of-line character.